contrib/build-push-ceph-container-imgs.sh

```diff
@@ -37,9 +39,11 @@ CONTAINER_REPO_PASSWORD=${CONTAINER_REPO_PASSWORD:-$DOCKER_HUB_PASSWORD}
 # GIT_BRANCH is typically 'origin/master', we strip the variable to only get 'master'
 CONTAINER_BRANCH="${GIT_BRANCH#*/}"
 CONTAINER_SHA=$(git rev-parse --short HEAD)
+OS=${OS:=centos}
```

Comment:
Again, this isn't useful especially when we have variables named with CENTOS or function like `_centos_release`

Reply:
since this script is move gravitated to make use of centos containers, I agree ... otherwise it would be a lot of work without any usecase for now. I will update my PR, thanks Dimitri!